docs/pseudo_cells.md

```diff
@@ -0,0 +1,36 @@
+## Pseudo Cells
+
+Pseudo PIPs and site pseudo PIPs are edges in the device graph that route
+through other sites and/or BELs.  Pseudo cells are the expression of the what
+routing resources are "blocked" by the use of a pseudo PIP.
+
+The device database currently expresses pseudo PIPs as using BEL pins within
+the site that the pseudo PIP is attached too.  Both input and output BEL pins
+are included in the pseudo cell definition, but only output BEL pins "block"
+the site wire.
+
+### Example
+
+In the case of a `CLBLL`'s `CLBLL_LL_A1` to `CLBLL_LL_A` pseudo PIP, this PIP
```

Comment:
We should probably specify the vendor and architecture for this example as we intend to support a variety of them.

Reply:
Fixed.